src/container/srv_target.c

```diff
@@ -170,24 +170,19 @@ cont_aggregate_runnable(struct ds_cont_child *cont, struct sched_request *req,
 		return false;
 	}
 
-	if (param->ap_vos_agg) {
-		/* Parse aggregation until reintegrating finish, because
-		 * vos_discard may cause issue if reintegration happened
-		 * at the same time.
-		 */
-		if (pool->sp_reintegrating) {
-			cont->sc_vos_agg_active = 0;
-			D_DEBUG(DB_EPC, DF_CONT": skip aggregation during reintegration %d.\n",
-				DP_CONT(cont->sc_pool->spc_uuid, cont->sc_uuid),
-				pool->sp_reintegrating);
-			return false;
-		}
-		if (!cont->sc_vos_agg_active)
-			D_DEBUG(DB_EPC, DF_CONT": resume aggregation after reintegration.\n",
-				DP_CONT(cont->sc_pool->spc_uuid, cont->sc_uuid));
-		cont->sc_vos_agg_active = 1;
+	if (pool->sp_reintegrating) {
+		cont->sc_agg_active = 0;
```

Comment:
so there are two ULTs, EC agg ULT and VOS agg ULT, how can single bit represents 2 ULTs running state? I think we need two bits here, right?

Reply:
right, just update, please check.